scripts/extensions/sams/src/components/assets/assetEditor.tsx

```diff
@@ -20,27 +22,119 @@ import {getHumanReadableFileSize} from '../../utils/ui';
 interface IProps {
     asset: Partial<IAssetItem>;
     disabled?: boolean;
-    onChange(field: string, value: string): void;
+    onChange/* <K extends keyof IAssetItem> */(field: string, value: any): void;
```

Comment:
What's wrong with the types on this function prop? i.e. can you un-comment so it's like this:
```typescript
onChange<K extends keyof IAssetItem>(field: K, value: IAssetItem[K]): void;
```

Reply:
Ohh, my bad. removing it.